src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+import duke.tasks.TaskList;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Represents the file containing the list of tasks.
+ */
+public class Storage {
+    private static final String text = "./data/duke.txt";
+    private static BufferedWriter writer;
+
+    /**
+     * Takes in a task and adds it to the tasklist, while writing it to the file
+     * @param task Task ot be added.
+     * @throws IOException if an error while reading has occurred.
+     */
+    static void writeToFile(String task) throws IOException {
+        try {
+            Parser.parseTask(task);
+            if (task.startsWith("deadline")
+                    || (task.startsWith("event") && task.contains("/at "))
+                    || (task.startsWith("todo"))) {
```

Comment:
Perhaps you could further unpack this into variables instead to simplify the code readablity and logic ?
for example: 
boolean isDeadline = task.startsWith("deadline");
...